share/ansible/roles/ci_run/tasks/alpine.yml

```diff
@@ -71,3 +71,22 @@
     src: /usr/local/src/shadow/etc/login.defs
     dest: /etc/login.defs
     remote_src: yes
+
+#The following is a workaround
+- name: Ensure test dependencies are installed
+  community.general.apk:
+    name:
+      - bash
+    state: present
+
+#The following is a workaround
+- name: Change shell for tests
+  ansible.builtin.command:
+    chsh -s /bin/bash
```

Comment:
Do we need to report a bug to pytest-mh?  I'd like to know that the workaround is temporary if there's a bug in a software we're using.

Reply:
I used the word workaround for lack of a better one. I was talking about this problem with @pbrezina (the pytest-mh maintainer) and this is the first time he encounters the problem of having to use another shell than bash, so the use of this shell is hardcoded. This by itself would not be a compelling enough reason to implement this change on our part, of course if other reasons are found or someone wants to implement this change I don't think there would be any problem.

Reply:
But code that needs bash mush use a bash shebang (`#!/usr/bin/env bash`) or something equivalent.  Why is the workaround needed?

Reply:
As for installing dependencies, how is pytest-mh installed?  Doesn't it pull dependencies as usual via the system package manager?  Why do we need to manually specify deps?

Reply:
> Ahh, yeah, that sounds more reasonable. So, the program fails if bash is not installed, and when bash is installed it works fine without needing to do anything special, right?

Yes, you are right, but this is because I force bash as a shell later on.

> As for installing dependencies, how is pytest-mh installed? Doesn't it pull dependencies as usual via the system package manager? Why do we need to manually specify deps?

Python dependencies are installed in a virtual environment using pip. They are listed in the [requirements](https://github.com/shadow-maint/shadow/pull/1131/files#diff-f3c0fe44e24c0b0d0ed6ce1b65a761c0ea590440d10644c7b1704516365fed60) file.